### PR TITLE
Sort helm versions docsgen

### PR DESCRIPTION
**Pull request by ashleywang1**

### CODE REVIEW

File: docs/docsgen/copy_helm_values_docs.go [hunk 1]
```diff
@@ -136,8 +136,8 @@ func copyHelmValuesDocsForComponent(
 		}
 		versions = append(versions, version)
 	}
-	sort.Reverse(semver.Collection(versions))
-	versions = getLatestPerMinorVersion(versions)
+	sort.Sort(sort.Reverse(semver.Collection(versions)))
```
Comment (joekelley) [inline docs/docsgen/copy_helm_values_docs.go:139]: do we need to reverse then sort? could we just sort?

Comment (ashleywang1) [reply docs/docsgen/copy_helm_values_docs.go:139]: Yep! We expect the results to be sorted from highest semver to lowest, wheras sort will return the versions from lowest to highest.

**Review by ryantking — APPROVED** [review]

File: docs/docsgen/copy_helm_values_docs.go (hunk 1 above)
Comment (harveyxia) [reply docs/docsgen/copy_helm_values_docs.go:139]: Elaborating on Ashley's explanation a bit—`sort.Reverse(...)` returns a `sort.Interface`, which requires an invocation of `sort.Sort` to actually perform the sorting mutation

**Pull request merged**